ape-config.yaml

```diff
@@ -23,13 +23,20 @@ solidity:
   version: 0.8.20
   evm_version: paris
   import_remapping:
-    - "@openzeppelin/contracts=openzeppelin/v4.8.1"
-    - "@openzeppelin-upgradeable/contracts=openzeppelin-upgradeable/v4.8.1"
+    - "@openzeppelin/contracts=openzeppelin/v4.9.1"
+    - "@openzeppelin-upgradeable/contracts=openzeppelin-upgradeable/v4.9.1"
     - "@fx-portal/contracts=fx-portal/v1.0.5"
 
 deployments:
   polygon:
+    mainnet:
+      - contract_type: DAI
```

Comment:
🚀

Reply:
Potentially. The deployment script for FlatRateFeeModel takes a  `currency` parameter address, but if you pass `DAI` it will look for the address here.